### PR TITLE
Remove xges adapter related files

### PR DESCRIPTION
**Pull request by rosborne132**
Part of issue: [```Rearrange PyPI projects```](https://github.com/AcademySoftwareFoundation/OpenTimelineIO/issues/1386)

**Summarize your change.**

- Removed xges adapter files.
- Files include: adapter code, tests, configs, and docs.

**Note:** No new tests were provided in this PR sense we are just moving the adapter code into its own repo.

### CODE REVIEW

File: contrib/opentimelineio_contrib/adapters/contrib_adapters.plugin_manifest.json [hunk 1]
```diff
@@ -1,18 +1,5 @@
...
-    "schemadefs" : [
-        {
-            "OTIO_SCHEMA" : "SchemaDef.1",
-            "name" : "xges",
-            "filepath" : "xges.py"
-        }
-    ]
+    "adapters": [],
+    "schemadefs" : []
 }
```
Comment (ssteinbach) [inline contrib/opentimelineio_contrib/adapters/contrib_adapters.plugin_manifest.json:5]: Wild to see this empty.  Nice work @apetrynet and @rosborne132!  After this PR is the next step  to remove the contrib area altogether?

**Review by apetrynet — APPROVED** [review]

**Comment by apetrynet** [comment]
It's a wrap @rosborne132! Thank you so much!

File: contrib/opentimelineio_contrib/adapters/contrib_adapters.plugin_manifest.json (hunk 1 above)
Comment (JeanChristopheMorinPerso) [reply contrib/opentimelineio_contrib/adapters/contrib_adapters.plugin_manifest.json:5]: It'll require some changes to https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/main/src/py-opentimelineio/opentimelineio/plugins/manifest.py#L214, but it should be pretty straight forward. Unless by system you mean ecosystem, in which case we'll have to test using all adapters installed in the same environment.

Comment (apetrynet) [reply contrib/opentimelineio_contrib/adapters/contrib_adapters.plugin_manifest.json:5]: I'll have to look a bit closer, but what I meant was making sure search paths and where external plugins register don't require the contrib folder. I think it should be fairly straight forward, yes.
We should create a separate issue for this.

**Pull request merged**